pkgs/utils/src/cast.zig

```diff
@@ -0,0 +1,15 @@
+const std = @import("std");
+
+pub fn Cast(To: type, from: anytype) To {
+    var result: To = undefined;
+    inline for (@typeInfo(To).@"struct".fields) |field| {
```

Comment:
might be worth ignoring fields if they don't exist in one or the other structure? or make it an option at least.

Reply:
I mean to also ignore fields that are missing in the from, but it's ok, let's implement it when we need it.